base/setup/usetup/usetup.c

```diff
@@ -143,6 +143,23 @@ DoWatchDestFileName(LPCWSTR FileName)
 
 /* FUNCTIONS ****************************************************************/
 
+VOID StopWatch(LPCSTR Name OPTIONAL)
```

Comment:
Debug code to be removed for final PR version.

Reply:
Deleted.